packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -338,7 +338,7 @@ export function toInstrumentedOdspTokenFetcher(
 export function createCacheSnapshotKey(odspResolvedUrl: IOdspResolvedUrl): ICacheEntry {
     const cacheEntry: ICacheEntry = {
         type: snapshotKey,
-        key: "",
+        key: odspResolvedUrl.fileVersion ?? "",
```

Comment:
That's probably sufficient, but I think we need to evaluate if hashedDocumentId should include file version as well in its calculus (outside of this PR)